### DISCUSSION TITLE
Arduino firmware not working for me

### DISCUSSION THREAD

**Discussion by lolomovil** [nofeletru/UsbAsp-flash#61] [Q&A]
I'm trying to use the software with arduino firmware and I cannot read an i2c 24LC16B eeprom chip. The software seems to work and communicate with the arduino but the reading obtained is all 0's. The wiring is fine since I can read the chip with a small Arduino sketch.

**Comment by nofeletru** [comment]
Dunno. There is no much feedback about arduino firmware. 
Maybe need pull-up resistors on  sda and scl pins?
